BaseApi/V1/Profiles/MappingProfile.cs

```diff
@@ -8,7 +8,7 @@ public class MappingProfile : Profile
     {
         public MappingProfile()
         {
-            CreateMap<DayCarePackageForCreationDto, DayCarePackage>();
+            CreateMap<DayCarePackageForCreationRequest, DayCarePackage>();
```

Comment:
I don't think there's a case where we would convert from boundary -> entity

We would work boundary -> domain -> entity

Reply:
That conversion would be one to one for every field from boundary to domain. That necessary?

Reply:
Yes it's part of the requirements we were given, auto mapper can achieve this too